Cargo.toml

```diff
@@ -348,13 +348,13 @@ webbrowser = "1.0"
 winit = { version = "0.30.5", default-features = false }
 # TODO(andreas): Try to get rid of `fragile-send-sync-non-atomic-wasm`. This requires re_renderer being aware of single-thread restriction on resources.
 # See also https://gpuweb.github.io/gpuweb/explainer/#multithreading-transfer (unsolved part of the Spec as of writing!)
-wgpu = { version = "24.0", default-features = false, features = [
+wgpu = { version = "25.0", default-features = false, features = [
   # Backends (see https://docs.rs/wgpu/latest/wgpu/#feature-flags)
   "webgl",
   "metal",
   "webgpu",
-  #"vulkan", # Does not yet exist, wgpu enables this automatically on Linux & Windows.
-  #"gl",     # Does not yet exist, wgpu enables this automatically on Linux & Windows.
+  "vulkan",
+  "gles",
```

Comment:
Are these just all the default features? Would it make sense to just use `default-features = true` instead?

Reply:
dx12 would be in the default set, we're leaving that out as per comment below